hax/hax/motr/__init__.py

```diff
@@ -195,10 +195,6 @@ def _process_event_cb(self, fid, chp_event, chp_type, chp_pid):
                               chp_type=chp_type,
                               chp_pid=chp_pid,
                               fid=fid)))
-        if chp_event == 3:
```

Comment:
I guess this check was added by @rajanikantchirmade.
Are you sure you don't need it?

Reply:
Yeah we don't need this now because this is happening automatically, hax notifies M0_NC_FAILED to motr process as hax receives notification from Consul regarding motr service state change in consul service catalog (failure/stop).